scripts/prepare_sector_network.py

```diff
@@ -4617,7 +5164,7 @@ def add_enhanced_geothermal(n, egs_potentials, egs_overlap, costs):
     options = snakemake.params.sector
     cf_industry = snakemake.params.industry
 
-    investment_year = int(snakemake.wildcards.planning_horizons)
```

Comment:
A rebase would be great since it makes reviewing a bit easier
Was a bit confused and had to check the pypsa-eur master